plottable.css

```diff
@@ -65,17 +65,17 @@ svg.plottable {
 }
 
 .plottable .label-area text {
-  fill: #32313F;
```

Comment:
The colors used here were selected by our designers; please don't change them. The recommended way to change the styles for your use case is to load a separate CSS file that overrides them.